relay-event-schema/src/protocol/contexts/trace.rs

```diff
@@ -138,6 +138,10 @@ pub struct TraceContext {
     #[metastructure(pii = "maybe", skip_serialization = "null")]
     pub data: Annotated<SpanData>,
 
+    /// Links to other spans from the trace's root span.
+    #[metastructure(pii = "maybe", skip_serialization = "null")]
```

Comment:
`pii = "maybe"` means it will only be scrubbed with Advanced Data Scrubbing Rules. Are there any link attributes that we expect to contain PII?

Reply:
None so far. I'm just saying given that attribute values are free text, they _could_